src/main/java/core/basesyntax/reportservices/ReportToFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.reportservices;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+public class ReportToFileImpl implements ReportToFile {
+
+    @Override
+    public void writeReportToFile(List<String> report, String fileName) {
+        try (PrintWriter writer = new PrintWriter(new FileWriter(fileName))) {
+            for (String line : report) {
+                writer.println(line);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Instead of printing the stack trace, rethrow a RuntimeException with an informative message. This will help in understanding what went wrong without exposing the internal details of the exception.